be/src/http/http_common.h

```diff
@@ -52,7 +52,11 @@ static const std::string HTTP_TRIM_SPACE = "trim_space";
 static const std::string HTTP_ENCLOSE = "enclose";
 static const std::string HTTP_ESCAPE = "escape";
 static const std::string HTTP_MAX_FILTER_RATIO = "max_filter_ratio";
+// For stream load, the timeout for transaction from PREPARE -> COMMITTED.
+// For transaction stream load, the timeout for transaction from PREPARE -> PREPARED.
```

Comment:
this comment is a little confused. What if the txn get timed out? automatically turn to next state or cancelled?